plenum/server/consensus/view_change_service.py

```diff
@@ -85,7 +87,16 @@ def process_need_view_change(self, msg: NeedViewChange):
         self._votes.add_view_change(vc, self._data.name)
 
         # 6. Unstash messages for new view
-        self._router.process_all_stashed()
+        self._router.process_all_stashed(STASH_VIEW)
+
+        # 7. Schedule New View after timeout
+        if self._resend_inst_change_timer is not None:
+            self._resend_inst_change_timer.stop()
+        self._resend_inst_change_timer = \
+            RepeatingTimer(self._timer,
```

Comment:
I think will be better in a future use one RepeatingTimer without recreating it.